.github/workflows/ci.yml

```diff
@@ -287,9 +287,11 @@ jobs:
 
           # check with no features
           cargo clippy --target ${{ matrix.target }} ${{ matrix.extra-flags }} -p wgpu-types --no-default-features
+          cargo clippy --target ${{ matrix.target }} ${{ matrix.extra-flags }} -p naga --no-default-features --features libm
 
           # Check with all features except "std".
           cargo clippy --target ${{ matrix.target }} ${{ matrix.extra-flags }} -p wgpu-types --no-default-features --features strict_asserts,fragile-send-sync-non-atomic-wasm,serde,counters
+          cargo clippy --target ${{ matrix.target }} ${{ matrix.extra-flags }} -p naga --no-default-features --features libm,dot-out,glsl-out,msl-out,msl-out-if-target-apple,wgsl-out,hlsl-out,hlsl-out-if-target-windows,compact
```

Comment:
I have concerns about this list atrophying, I'm not sure what should be done about that though.